apps/webview/src/components/protectedRoute.tsx

```diff
@@ -0,0 +1,23 @@
+import { Navigate } from 'react-router-dom';
```

Comment:
Look into the use of ```<Outlet />``` component from react-router-dom here. It simplifies routing in protected routes.